pom.xml

```diff
@@ -336,13 +336,13 @@ Require-Capability: osgi.extender;filter:="(&(osgi.extender=sling.scripting)(ver
         <dependency>
             <groupId>org.apache.sling</groupId>
             <artifactId>org.apache.sling.scripting.sightly.runtime</artifactId>
-            <version>1.2.4-1.4.0</version>
+            <version>1.2.5-1.4.0-SNAPSHOT</version>
             <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>org.apache.sling</groupId>
             <artifactId>org.apache.sling.scripting.sightly.compiler</artifactId>
-            <version>1.2.12-1.4.0</version>
+            <version>1.2.13-1.4.0-SNAPSHOT</version>
```

Comment:
I don't think you need to bump the compiler.

Reply:
Ah, that's for https://github.com/apache/sling-org-apache-sling-scripting-sightly-compiler/pull/8, but it was not clear from this PR, nor from https://issues.apache.org/jira/browse/SLING-10677.

Reply:
> I don't think you need to bump the compiler.

You are probably right.  None of the tests here are directly impacted by https://github.com/apache/sling-org-apache-sling-scripting-sightly-compiler/pull/8 

The testing for SLING-10682 was done with local unit tests.